### DISCUSSION TITLE
Inconsistent replay playback for sliderticks & sliderends

### DISCUSSION THREAD

**Discussion by sz-da** [ppy/osu#26948] [Q&A]
When playing back a replay, whether a slidertick/sliderend shows as hit or miss can change randomly.

Interesting thing I noticed while investigating this issue:
Unlike stable, lazer replays don't contain a replay frame for sliderticks or sliderends.

## Slidertick example:

https://github.com/ppy/osu/assets/76718358/51178818-072f-4e9f-9d27-7afd9d21d0f8

Replay: https://drive.google.com/uc?id=1amOQRZyuzyOqJhyWdkkzn6UtuzEVvN1l
Time at which it occurs: 61021ms

Relevant replay frames:
| Time (ms) | x | y | keys pressed |
| ----------- | - | - | -------------- |
| 61018 | 248.44446 | 66.66667 | M1 |
| 61039 | 250.66669 | 68.88889 | M1 |

## Sliderend example:

https://github.com/ppy/osu/assets/76718358/e53c2a1f-faaf-49d2-8f4b-046826fa1903

Replay: https://drive.google.com/uc?id=1_7z2HqmFqjfn6eFM7ZQ-nEQgAJyirnjf
Time at which it occurs: 62135ms

Relevant replay frames:
| Time (ms) | x | y | keys pressed |
| ----------- | - | - | -------------- |
| 62110 | 371.55554 | 154.22223 | M2 |
| 62130 | 370.22223 | 121.77779 | M2 |
| 62151 | 376.0 | 99.111115 | M2 |
| 62168 | 380.44446 | 87.111115 | M2 |
| 62190 | 384.88892 | 83.111115 | M2 |

**Comment by frenzibyte** [comment]
Potentially related: https://github.com/ppy/osu/issues/21659
